tools/cmake/component.cmake

```diff
@@ -376,10 +376,17 @@ endmacro()
 function(idf_component_get_property var component property)
     cmake_parse_arguments(_ "GENERATOR_EXPRESSION" "" "" ${ARGN})
     __component_get_target(component_target ${component})
-    if(__GENERATOR_EXPRESSION)
-        set(val "$<TARGET_PROPERTY:${component_target},${property}>")
+    if("${component_target}" STREQUAL "")
+        if("${component}" STREQUAL "esp-wolfssl")
+            message(STATUS "Component esp-wolfssl needs to be installed. See api-reference/protocols/esp_tls docs.")
```

Comment:
I would prefer to not add handling of specific components in `tools/cmake`.

Please check for presence of `esp-wolfssl` component before calling idf_component_get_property(esp-wolfssl ...), instead.

Reply:
Ah ok. I misunderstood your [suggestion](https://github.com/espressif/esp-idf/pull/14033#discussion_r1647368051). I've updated this PR for the `idf_component_get_property` change.

I have a much more elaborate check WIP in [components/esp-tls/CMakeLists.txt](https://github.com/gojimmypi/esp-idf/blob/ac2e1f572d8b3c7c4fb3664c9fc7a6be2e04b79e/components/esp-tls/CMakeLists.txt#L65) regarding the wolfSSL component.